api/data_refinery_api/views/compendium_result.py

```diff
@@ -125,7 +123,8 @@ def get_queryset(self):
     def get_serializer_class(self):
         try:
             token_id = self.request.META.get("HTTP_API_KEY", None)
-            token = APIToken.objects.get(id=token_id, is_activated=True)
+            # Verify that a token with that id exists
+            _ = APIToken.objects.get(id=token_id, is_activated=True)
```

Comment:
Honestly we probably don't even need the assignment at all, but this is fine.

Reply:
If you don't need the assignment, just ditch it :)

Reply:
I see this pattern in a few places. I much prefer just doing the function call instead of having an unused assignment.